xformers/factory/block_factory.py

```diff
@@ -335,8 +335,8 @@ def forward(
             q, k, v = x, x, x
 
         # Pre/Post norms and residual paths are already handled
-        x = self.wrap_att(q, k, v, att_mask=att_mask)
-        x = self.wrap_ff(x)
+        x = self.wrap_att(inputs=[q, k, v], att_mask=att_mask)
```

Comment:
all the wraps require a single input list + kwargs, which I believe is more future proof (this bug cannot happen, or at least not as easily)